openapi/xero.assets/client.bal

```diff
@@ -0,0 +1,160 @@
+import  ballerina/http;
```

Comment:
Licence header?

Reply:
Added. Seems License header not get added

Reply:
Added